src/tools/rbd/ArgumentTypes.cc

```diff
@@ -130,6 +130,26 @@ void add_image_option(po::options_description *opt,
     (name.c_str(), po::value<std::string>(), description.c_str());
 }
 
+void add_group_option(po::options_description *opt,
+		      ArgumentModifier modifier,
+		      const std::string &desc_suffix) {
+  std::string name = GROUP_NAME;
+  std::string description = "group name";
+  switch (modifier) {
+  case ARGUMENT_MODIFIER_NONE:
+  case ARGUMENT_MODIFIER_SOURCE:
+    break;
+  case ARGUMENT_MODIFIER_DEST:
+    name = DEST_GROUP_NAME;
```

Comment:
Why does this require a dest-group argument?

Reply:
I've made changes now so that `tools/rbd/action/Group.cc` also uses this `add_group_option()`  in [ArgumentTypes.cc](https://github.com/ceph/ceph/pull/60627/commits/bb6bd71df0df93fa25fc753ab1bbe0383703a3b5#diff-f8c5882c91c1cf5eee8749346a253a7fb007531c3c202c78b176ac9e05c4131a) .  So the  dest-group argument will be used in group rename command